### PR TITLE
DOC: escaping code examples properly in read_csv docstring

### LINKED ISSUE #18411: Newline backslash stripped
In `read_csv` [documentation][0], there is an example: 

> For example, if comment=’#’, parsing ‘#emptyna,b,cn1,2,3’ with header=0 will result in ‘a,b,c’ being treated as the header.

Newlines have their backslashes stripped, because these examples are not encapsulated by double backticks.

Will submit a PR.

[0]: https://pandas.pydata.org/pandas-docs/stable/generated/pandas.read_csv.html

### PR DESCRIPTION
**Pull request by kokes**
- [x] closes #18411
~~- [ ] tests added / passed~~
~~- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`~~
~~- [ ] whatsnew entry~~

### CODE REVIEW

**Comment by jreback** [comment]
@TomAugspurger @jorisvandenbossche

File: pandas/io/parsers.py [hunk 1]
```diff
@@ -245,9 +245,9 @@
     Indicates remainder of line should not be parsed. If found at the beginning
     of a line, the line will be ignored altogether. This parameter must be a
     single character. Like empty lines (as long as ``skip_blank_lines=True``),
-    fully commented lines are ignored by the parameter `header` but not by
-    `skiprows`. For example, if comment='#', parsing '#empty\\na,b,c\\n1,2,3'
-    with `header=0` will result in 'a,b,c' being
+    fully commented lines are ignored by the parameter ``header`` but not by
```
Comment (jorisvandenbossche) [inline pandas/io/parsers.py]: Can you leave 'header' (and skiprows below) with single backticks? (that's how literal references to kwargs are done in the rest of the docstring I think). 
For the code example, the double ones are good!

Comment (kokes) [reply pandas/io/parsers.py]: yup, done

**Comment by jorisvandenbossche** [comment]
Thanks!

**Comment by jorisvandenbossche** [comment]
Can you check http://pandas-docs.github.io/pandas-docs-travis/generated/pandas.read_csv.html#pandas.read_csv in a few hours to see if it now is correctly displaying?

**Comment by kokes** [comment]
Will do, just set a reminder. I checked it locally, but will take a look at the resulting production documentation later today.

**Pull request merged**